great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -306,57 +303,58 @@ def get_expectation_suite(
     ) -> ExpectationSuite:
         """
         Args:
-            expectation_suite: An existing "ExpectationSuite" to update.
-            expectation_suite_name: A name for returned "ExpectationSuite".
+            expectation_suite: An existing "ExpectationSuite" to update
+            expectation_suite_name: A name for returned "ExpectationSuite"
             include_citation: Whether or not to include the Profiler config in the metadata for "ExpectationSuite" produced by "RuleBasedProfiler"
 
         Returns:
-            "ExpectationSuite" using "ExpectationConfiguration" objects, computed by "RuleBasedProfiler" state.
+            "ExpectationSuite" using "ExpectationConfiguration" objects, computed by "RuleBasedProfiler" state
         """
         return self.profiler.get_expectation_suite(
             expectation_suite=expectation_suite,
             expectation_suite_name=expectation_suite_name,
             include_citation=include_citation,
         )
 
-
```

Comment:
@NathanFarmer I am unable to see exactly what is being done differently here -- perhaps we can review together.  Thanks!

Reply:
@alexsherstinsky I changed it and then changed my mind but forgot to change it back. I have reverted this change. ✅